frontend/src/v5/store/tickets/tickets.types.ts

```diff
@@ -24,7 +24,9 @@ export interface PropertyDefinition {
 	type: PropertyTypeDefinition;
 	values?: string[] | 'jobsAndUsers' | 'riskCategories';
 	default?: string;
+	unique?: boolean;
```

Comment:
When there's a unique error triggered, after closing the modal, the save ticket button stays in processing state.
![cant edit](https://github.com/3drepo/3drepo.io/assets/42032433/173b1524-a081-497c-b6bb-315775e6070f)